panel/viewable.py

```diff
@@ -1149,10 +1149,16 @@ def __init__(
     def _transform_value(self, val):
         if isinstance(val, list) and val:
             from .pane import panel
-            val[:] = [
-                v if isinstance(v, Viewable) else panel(v)
-                for v in val
-            ]
+            new = []
+            mutated = False
+            for v in val:
+                if not isinstance(v, Viewable):
+                    n = panel(v)
+                    mutated |= v is not n
```

Comment:
Out of curiosity, what kind of object that is not a `Viewable` would calling `panel(obj)` on return itself?

Reply:
Good point, it shouldn't return anything else. Will leave this just in case.